Makefile

```diff
@@ -558,8 +566,25 @@ check-symbols: startup_files libc
 	# This ignores whitespace because on Windows the output has CRLF line endings.
 	diff -wur "$(CURDIR)/expected/$(MULTIARCH_TRIPLE)" "$(SYSROOT_SHARE)"
 
+NORMAL_SYSROOT := \
+	$(findstring $(SYSROOT)/lib,$(SYSROOT_LIB)) \
+	$(findstring $(SYSROOT)/include,$(SYSROOT_INC)) \
+	$(findstring $(SYSROOT)/share,$(SYSROOT_SHARE))
+
+# If the 3 dirs are not within the standard locations, the install rule
+# is undefined. This protects the user from doing something silly by
+# mistake.
+ifeq ($(SYSROOT)/lib $(SYSROOT)/include $(SYSROOT)/share,$(strip $(NORMAL_SYSROOT)))
 install: finish
 	mkdir -p "$(INSTALL_DIR)"
 	cp -r "$(SYSROOT)/lib" "$(SYSROOT)/share" "$(SYSROOT)/include" "$(INSTALL_DIR)"
+.PHONY: install
+endif
+
+clean:
+	$(RM) -r "$(OBJDIR)"
+	$(RM) -r "$(SYSROOT_INC)"
+	$(RM) -r "$(SYSROOT_LIB)"
+	$(RM) -r "$(SYSROOT_SHARE)"
```

Comment:
@sbc100 I added a `make clean` rule, so there is still an easy and standard way to clean up after everything.